lib/phantomjs/platform.rb

```diff
@@ -114,7 +114,7 @@ def platform
         end
 
         def package_url
-          'https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-linux-i686.tar.bz2'
+          'https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.0.0-source.zip'
```

Comment:
This gem doesn't have support for installing from source, and there will not be precompiled binaries made available for 2.0.0.